src/offline/cable_driver_init.F90

```diff
@@ -84,38 +81,25 @@ MODULE cable_driver_init_mod
 
 CONTAINS
 
-  SUBROUTINE cable_driver_init()
+  SUBROUTINE cable_driver_init(mpi_grp)
+    TYPE(mpi_grp_t), INTENT(IN) :: mpi_grp
+
     !! Model initialisation routine for the CABLE offline driver.
```

Comment:
Please describe `mpi_grp` in the ford docstring. See https://cable--469.org.readthedocs.build/en/469/api/proc/cable_driver_init.html for preview